src/app/services/provisioning-helper/provisioning-helper.coffee

```diff
@@ -10,4 +10,7 @@ angular.module 'mnoEnterpriseAngular'
         (pp) -> !_self.pricedPlan(pp) || _.some(pp.prices, (p) -> p.currency == currency)
       )
 
+    @skipPricingPlans = (product) ->
+      product.product_type == 'application' && (!product.single_billing_enabled || !product.billed_locally)
```

Comment:
@paulstuartparker @iseessel @adamaziz15 @yuritomanek (sorry for the spam)

After discussion with @BrunoChauvet, I think we shouldn't not skip the pricing plan step when then product is billed externally. We should only skip it when the app doesn't support single billing.

See OPALSUP-187

Reply:
You should actually skip the pricing selection if Single Billing is disabled AND Provisioning is disabled
We could face the scenario where an application is provisioned externally but and the billing is managed by the application. In this case we still need to ask the user to select a pricing plan to pass down to the external system.

Reply:
It's the `Product#externally_provisioned` flag. In this case, even if there's no single billing we still need to tell the external application which pricing plan they need to provision.

New behaviour, skip pricing selection for *applications* when:
* `product.single_billing_enabled == false` AND `product.externally_provisioned` == false

It might be easier to redefine it this way to avoid double negation:
```coffeescript
@skipPricingPlans = (product) ->
  product.product_type == 'application' && !product.single_billing_enabled && !product.externally_provisioned

# Or if you want to avoid too much negation
@displayPricingPlans = (product) ->
  product.product_type != 'application' || product.single_billing_enabled || product.externally_provisioned

```

@yuritomanek  @BrunoChauvet Do you agree?